tests-perf/locust/send_rate_sms.py

```diff
@@ -0,0 +1,58 @@
+# isort:skip_file
+# flake8: noqa
+
+from tests_smoke.smoke.common import job_line, rows_to_csv  # type: ignore
+from locust import HttpUser, constant_pacing, task
+from dotenv import load_dotenv
+from dataclasses import make_dataclass
+from datetime import datetime
+import sys
+import os
+BULK_SIZE = 2000
+
+
+sys.path.append(os.path.abspath(os.path.join("..", "tests_smoke")))
+
+
+load_dotenv()
+NotifyApiUserTemplateGroup = make_dataclass('NotifyApiUserTemplateGroup', [
```

Comment:
You can also create a "regular" class annotated with `@dataclass` which might be more naturally looking to Python eyes.